charts/geoserver/README.md

```diff
@@ -0,0 +1,96 @@
+# geoserver
+A simple Helm chart for GeoServer
+
+
+## TL;DR;
+
+```bash
+$ helm repo add ncsa https://opensource.ncsa.illinois.edu/charts/
+$ helm install geoserver ncsa/geoserver
+```
+
+## Prerequisites
+
+- Kubernetes 1.16+
+- helm 3
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release ncsa/geoserver
+```
+
+The command deploys GeoServer on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+Below are the supported configuration options that can be overridden or customized in `values.yaml`:
+
+| Key | Description | Default Value |
+| --- | --- | --- |
+| `persistence.accessModes` | The access mode with which to mount elasticsearch data | `ReadWriteOnce` |
+| `persistence.storageClass` | The storage class to use to provision the elasticsearch data PVC | (empty - use cluster default) |
+| `persistence.capacity` | The size of the PVC to provision for elasticsearch data | `10Gi` |
+| `persistence.existingClaim` | An existing claim to use | None |
+| `image.repository` | The Docker image repo/name to run | `docker.osgeo.org/geoserver` |
+| `image.tag` | The Docker image tag to run, leave blank to use app version | `""` |
+| `image.pullPolicy` | The Docker image pullPolicy to use when running | `IfNotPresent` |
+| `ingress.enabled` | Whether an ingress rule should be deployed for geoserver | `true` |
+| `ingress.host` | The hostname to use for the ingress rule | `geoserver.local` |
+| `resources` | [Resource limits](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) that should be applied to elasticsearch instances | None |
+| `nodeSelector` | [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector) that should be applied to elasticsearch instances | None |
+| `tolerations` | [Tolerations](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) that should be applied to elasticsearch instances | None |
+| `affinity` | [Affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) that should be applied to elasticsearch instances | None |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install --name my-release \
```

Comment:
we should remove the clowder references here.

Reply:
removed clowder part